nncf/tensorflow/helpers/model_creation.py

```diff
@@ -94,13 +98,19 @@ def create_compressed_model(model: tf.keras.Model,
             original_model_accuracy = evaluation_args.eval_fn(model)
 
     builder = create_compression_algorithm_builder(config, should_init=not compression_state)
+
+    NNCFTelemetry.send_event(event_category=NNCF_TF_CATEGORY,
+                             event_action='compression_started',
```

Comment:
Will it be duplicated by the same action on `apply` for experimental quantization?
https://github.com/openvinotoolkit/nncf/pull/1339/files#diff-37d0c21a327711cf378c2ddbbbbbb63bed3343b92fba312427a9e840bd42fc9eR113